quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/LoggingPropertyMappers.java

```diff
@@ -411,4 +420,35 @@ private static void validateSyslogMaxLength(String value) {
             throw new PropertyException(String.format("Invalid value for option '--log-syslog-max-length': %s", e.getMessage()));
         }
     }
+
+    // Workaround BEGIN - for https://github.com/keycloak/keycloak/issues/39893
+    // Remove once the https://github.com/quarkusio/quarkus/issues/48036 is included in Keycloak as Quarkus might handle it on its own
+    private static String resolveSyslogCountingFraming(String value, ConfigSourceInterceptorContext context) {
+        if (SYSLOG_COUNTING_FRAMING_PROTOCOL_DEPENDENT.equals(value)) {
+            return Configuration.getOptionalKcValue(LoggingOptions.LOG_SYSLOG_PROTOCOL)
+                    .map(LoggingPropertyMappers::handleSyslogCountingFramingBasedOnProtocol)
+                    .orElse(Boolean.FALSE.toString());
+        }
+        return value;
+    }
+
+    private static String resolveSyslogCountingFramingMapFrom(String protocol, ConfigSourceInterceptorContext context) {
```

Comment:
Is this needed? Since there is a default value it will be used and transformed without a need to do a mapFrom.

Reply:
You're right, thanks :)